lib/services/devices/deviceService.js

```diff
@@ -1228,6 +1228,7 @@ function retrieveDevice(deviceId, apiKey, callback) {
 exports.listDevices = intoTrans(context, checkRegistry)(listDevices);
 exports.listDevicesWithType = intoTrans(context, checkRegistry)(listDevicesWithType);
 exports.getDevice = intoTrans(context, checkRegistry)(getDevice);
+exports.getDeviceSilently = intoTrans(context, checkRegistry)(getDeviceSilently);
```

Comment:
No CNR entry needed?

Reply:
No CNR update is needed, it's just reopen https://github.com/telefonicaid/iotagent-node-lib/pull/891

Reply:
NTC